### ISSUE TITLE
Sales reports report type SALES freq DAILY requires version 1_1

### ISSUE THREAD

**Issue by Ella6882** [Ponytech/appstoreconnectapi#62]
There is a bug being discussed here (https://forums.developer.apple.com/forums/thread/745052) regarding this error:

```
appstoreconnect.api.APIError: The version parameter you have specified is invalid. The latest version for this report is 1_1.
```

when attempting to download the ```download_sales_and_trends_reports``` with a frequency of DAILY. 

It looks like the API versions are hard-coded in this repo.

```
	def download_sales_and_trends_reports(self, filters=None, save_to=None):
		# setup required filters if not provided
		default_versions = {
			'SALES': '1_0',...
		}
```

https://github.com/Ponytech/appstoreconnectapi/blob/master/appstoreconnect/api.py#L668

If we could get this upgraded to 1_1 that would be fantastic.

**Comment by SueeiiiZhu** [comment]
Can you use the 1_1 version of the API to get the last two days of sales data?
ERROR：appstoreconnect.api.APIError: There were no sales for the date specified.

**Comment by dsaxton-1password** [comment]
It's possible that removing the version filter will fix the issue, according to this thread: https://forums.developer.apple.com/forums/thread/745052

**Comment by SueeiiiZhu** [comment]
Yes, The version number was modified back by Apple and became "1_0"
